word_scripts/wordfind_site.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import word_file_merger
+from bs4 import BeautifulSoup
+
+def extract(htmlList):
+    words = []
+    
+    finding_words = list(filter(lambda x: x.get('class') == ['defLink'], htmlList))
+    
+    for link in finding_words:
+        words.append(link.find('a').get('href')[6:-1])     
+        
+    return words
+
+def extract_from_word_find_site():
+    wordfind_dict = {}
+    for word_length in range(3,12):
+        try:
+            req = requests.get("http://www.wordfind.com/{0}-letter-words/".format(word_length))
+            unextracted = BeautifulSoup(req.text, 'html.parser').find_all('li') 
+            wordFind_dict[word_length] = extract(unextracted)  
+        except requests.exceptions.ConnectionError as e:
+            print ("A connection error occurred.") 
+        
+    return wordfind_dict  
+
+word_file_merger.word_merger(extract_from_word_find_site())
```

Comment:
This is not very readable

Instead, at the top of the file `from word_file_merger import word_merger` and then you can just do `word_merger(...)`